support/@trustfractal/polkadot-utils/examples/high_throughput_tranfers.js

```diff
@@ -49,15 +49,15 @@ async function main() {
         const watcher =
             batchers[remaining % batchers.length].signAndSend(txn, alice);
         watchers.add(watcher);
-        await watcher.inBlock();
+        return await watcher.inBlock();
       })());
     }, 10)
   })
 
-  await Promise.all(promises);
-  console.log('All txns in block');
-  await Promise.all(Array.from(watchers).map(w => w.finalized()));
-  console.log('All txns finalized');
+  let inBlocks = await Promise.all(promises);
+  console.log('All txns in block', inBlocks);
```

Comment:
I think this will be a really long list of repeated block hashes. Probably better as a map between block-hash and number in block? Similar to how we do statuses above.